guidelines/terms/20/contrast-ratio.html

```diff
@@ -15,20 +15,20 @@
    
    <p class="note">Contrast ratios can range from 1 to 21 (commonly written 1:1 to 21:1).</p>
    
-   <p class="note">Because authors do not have control over user settings as to how text is rendered
-      (for example font smoothing or anti-aliasing), the contrast ratio for text can be
+   <p class="note">Because authors do not have control over user settings as to how text and non-text content is rendered
+      (for example font smoothing or anti-aliasing), the contrast ratio can be
       evaluated with anti-aliasing turned off.
```

Comment:
while the sentiment/idea is valid - that it's not just text but also non-text content, it's not possible to turn off anti-aliaising for things like SVG or CSS lines etc. And even for text, it's not always possible - depending on OS or user agent - to turn it off even for text rendering.

Reply:
instead of just "with anti-aliasing turned off", this may need to be expanded to something like 
```suggestion
      evaluated with anti-aliasing turned off or ignored.
```